src/api/libcellml/analysermodel.h

```diff
@@ -0,0 +1,452 @@
+     */
+    AnalyserVariablePtr variable(size_t index) const;
+
+    /**
+     * @brief Get the number of equations in the @c Model.
+     *
+     * Return the number of equations in the @c Model processed by this
+     * @c Analyser.
+     *
+     * @return The number of equations.
+     */
+    size_t equationCount() const;
+
+    /**
+     * @brief Get the equations in the @c Model.
+     *
+     * Return the equations in the @c Model processed by this @c Analyser.
+     *
+     * @return The equations.
+     */
+    std::vector<AnalyserEquationPtr> equations() const;
+
+    /**
+     * @brief Get the equation at @p index.
+     *
+     * Return the equation at the index @p index for the @c Model processed by
+     * this @c Analyser.
+     *
+     * @param index The index of the equation to return.
+     */
+    AnalyserEquationPtr equation(size_t index) const;
+
+    /**
+     * @brief Test to determine if @c AnalyserModel needs the equality function.
+     *
+     * Test to determine if @c AnalyserModel needs the equality function, return
+     * @c true if it does and @c false otherwise.
+     *
+     * @return @c true if @c AnalyserModel needs the equality function, @c false
+     * otherwise.
+     */
+    bool needEqFunction() const;
+
+    /**
+     * @brief Test to determine if @c AnalyserModel needs the inequality
+     * function.
+     *
+     * Test to determine if @c AnalyserModel needs the inequality function,
+     * return @c true if it does and @c false otherwise.
+     *
+     * @return @c true if @c AnalyserModel needs the inequality function,
+     * @c false otherwise.
+     */
+    bool needNeqFunction() const;
+
+    /**
+     * @brief Test to determine if @c AnalyserModel needs the lower than
+     * function.
+     *
+     * Test to determine if @c AnalyserModel needs the lower than function,
+     * return @c true if it does and @c false otherwise.
+     *
+     * @return @c true if @c AnalyserModel needs the lower than function,
+     * @c false otherwise.
+     */
+    bool needLtFunction() const;
+
+    /**
+     * @brief Test to determine if @c AnalyserModel needs the lower or equal
+     * than function.
+     *
+     * Test to determine if @c AnalyserModel needs the lower or equal than
+     * function, return @c true if it does and @c false otherwise.
+     *
+     * @return @c true if @c AnalyserModel needs the lower or equal than
+     * function, @c false otherwise.
+     */
+    bool needLeqFunction() const;
+
+    /**
+     * @brief Test to determine if @c AnalyserModel needs the greater than
+     * function.
+     *
+     * Test to determine if @c AnalyserModel needs the greater than function,
+     * return @c true if it does and @c false otherwise.
+     *
+     * @return @c true if @c AnalyserModel needs the greater than function,
+     * @c false otherwise.
+     */
+    bool needGtFunction() const;
+
+    /**
+     * @brief Test to determine if @c AnalyserModel needs the greater or equal
```

Comment:
"greater than or equal to" ("equal than" doesn't make sense).

Reply:
Agreed, copying/pasting... Fixed in commit e2600fb.